Ease.XUnit.Unity/XUnitUnityContainerTestBase.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Ease.Unity;
+
+namespace Ease.XUnit.Unity
+{
+    public abstract class XUnitUnityContainerTestBase : UnityContainerTestBase
+    {
+        // Added for consistency between testing frameworks, do we need this for xUnit?
```

Comment:
I believe you are correct. It is not required since XUnit runs ctor for each test.. but for now let's leave it for consistency